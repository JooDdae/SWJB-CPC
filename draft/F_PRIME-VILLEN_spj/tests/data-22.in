8111682769831789495786811861465125491213325731696091532388495701266316329177039846118317309941583952333673374253229353871579354196758133539527999076679977415341972075987673747231361582156893067259334939199689701320927892351994922034021508783111531540789291759490995471519349121776196132539592391332213817165380599439874123891049964920038221665370198819729767913491859951131831946347935563
