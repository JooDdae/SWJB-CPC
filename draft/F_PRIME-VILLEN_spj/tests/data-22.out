184
141 0 878 898 0 4 24 663 1081 284 629 1 3 94 1 0 2 2 1 3 4 449 795 3 2 1 2 1103 3 126 386 38 10 0 4 905 1058 0 23 0 932 160 3 146 3 1 2 2 2 19 2 2 4 583 1 10 2 710 0 3 22 2 3 81 19 763 495 3 1 22 1222 861 25 4 13 3 2 81 920 783 19 2 4 15 1 2 0 2 18 3 142 749 439 1 17 2 70 2 157 1196 126 454 1 4 24 1 2 3 0 1227 328 556 680 23 0 0 0 3 2 0 713 24 10 0 4 17 656 1182 0 3 8 70 0 1 0 4 4 114 18 2 371 17 52 0 2 2 1 1 221 4 0 119 1013 1170 1090 1 2 24 176 1192 304 1031 857 903 1098 4 10 4 874 70 0 1071 3 0 0 2 0 23 8 90 92 2 3 103 